File: solutions/javascript/line-up/1/line-up.js
export const format = (name, number) => {
  let lastdigit = number % 10;
  let lastwo = number % 100;
  let no_str = number.toString();
  if (lastwo==11 || lastwo==12 || lastwo==13){
    no_str = no_str + "th";
  }
  else {
    switch(lastdigit) {
      case 1: no_str += "st";
        break;
      case 2: no_str += "nd";
        break;
      case 3: no_str += "rd";
        break;
      default:
        no_str += "th";
    }
  }
  return name + ", you are the " + no_str + " customer we serve today. Thank you!"
};

File: solutions/csharp/line-up/1/LineUp.cs
public static class LineUp
{
    public static string Format(string name, int number)
    {
        int final_digit = number % 10;
        string suffix;
        switch(final_digit)
        {
            case 1: 
                suffix = number%100==11 ? "th" : "st";
                break;
            case 2:
                suffix = number%100==12 ? "th" : "nd";
                break;
            case 3:
                suffix = number%100==13 ? "th" : "rd";
                break;
            default:
                suffix = "th";
                break;
        }
       return name + ", you are the " + number.ToString() + suffix+" customer we serve today. Thank you!";
    }
}
